app/helpers/ukims_helper.rb

```diff
@@ -3,24 +3,79 @@ module UkimsHelper
   CUT_OFF_DATE = Date.new(2023, 9, 30)
 
   def uk_internal_market_scheme
-    if after_cut_off_date
+    if after_cut_off_date?
       'UK Internal Market Scheme'
     else
       'UK Trader Scheme'
     end
   end
 
   def ukims_annual_turnover
-    if after_cut_off_date
+    if after_cut_off_date?
       '£2,000,000'
     else
       '£500,000'
     end
   end
 
+  def ukims_application_link
+    if after_cut_off_date?
+      'https://www.gov.uk/guidance/apply-for-authorisation-for-the-uk-internal-market-scheme-if-you-bring-goods-into-northern-ireland'
+    else
+      'https://www.gov.uk/guidance/apply-for-authorisation-for-the-uk-trader-scheme-if-you-bring-goods-into-northern-ireland-from-1-january-2021'
+    end
+  end
+
+  def ukims_info_url
+    if after_cut_off_date?
+      'https://www.gov.uk/government/news/windsor-framework-unveiled-to-fix-problems-of-the-northern-ireland-protocol'
+    else
+      'https://www.gov.uk/guidance/apply-for-authorisation-for-the-uk-trader-scheme-if-you-bring-goods-into-northern-ireland'
+    end
+  end
+
+  def explore_topic_title
+    if after_cut_off_date?
+      'Find out more about the Windsor Framework'
+    else
+      'Explore the topic'
+    end
+  end
+
+  concerning :PlannedProcessingText do
+    def commercial_processing_hint_text
+      html_data = if after_cut_off_date?
+                    <<-PARAGRAPH
```

Comment:
Wondering whether it would be cleaner to just chuck these into a conditionally selected partial?

Reply:
I guess that is an option although this is included in the locales yml file and could get a bit messy either way. At least this way once a certain date has passed then we can just delete this code.

Reply:
This should sort out the indentation madness, too :tada:

Reply:
But we have an idiom/way of doing things and even if you delete the code you'd be diverging from that just for this specific case?

Reply:
Yeah il have a look at this, that does look better.

Reply:
The indentation was kind of on purpose but rubocop also didnt help lol

Reply:
[shakes fist at rubocop]